ecl/hql/hqlcollect.cpp

```diff
@@ -548,10 +548,16 @@ void FileSystemDirectory::processDependencies(IPropertyTree * dependTree, const
     {
         IPropertyTree & cur = depends->query();
         const char * name = cur.queryName();
+
         // The special node __empty__ is added for an emptry string mapping "" =>, which is the entry for this package
-        if (isEmptyString(name) || strieq(name, "__empty__"))
+        if (isEmptyString(name) || strieq(name, "__empty__") || strieq(name, "_0"))
```

Comment:
This seems like something we'd be using in lots of different places.  I'm wondering if this should be jlib library function:

isXmlValueEmpty(name) or isYamlValueEmpty(name)

Reply:
Now tony's PR is merged the middle entry can be removed.  I could add a function to hide the details.